.github/workflows/platform.deployment.history.removal.yml

```diff
@@ -1,4 +1,4 @@
-name: '.Platform: Remove Deployments'
+name: '.Platform: Remove Deployment History'
```

Comment:
Same as above, plus we should rename the 2 files

Reply:
Renamed